content/en/docs/release-notes/release-notes-1.2.md

```diff
@@ -5,14 +5,84 @@ weight: 810
-   Please set the `Certificate.Spec.RenewBefore` field instead.
-   This flag will be removed in the next release.
+Please read the [upgrade notes](/docs/installation/upgrading/upgrading-1.1-1.2/) before upgrading.
 
-1. In an effort to introduce new features whilst keeping the project
-   maintainable, cert-manager now only supports Kubernetes down to version
-   `v1.16`. This means the `legacy` manifests have now been removed. You can
-   read more [here](../../concepts/project-maturity/).
+Aside from that, there have been numerous bug fixes and features summarized below. 
+
+# Deprecated Features and Breaking Changes
+
+1. The `--renew-before-expiration-duration` flag of the cert-manager controller-manager has been deprecated. Please set the `Certificate.Spec.RenewBefore` field instead. This flag will be removed in the next release.
+
+2. As Kubernetes `v1.16` is now the earliest supported version, The `legacy` manifests have now been removed. You can read more [here](../../concepts/project-maturity/).
+
+3. The `User-Agent` request header has been changed from `jetstack-cert-manager/<version>` to `cert-manager/<version>`. This may affect functionality if you rely on an a User-Agent allowlist in a corporate environment.
+
+# Copyright and Ownership
+
+* As this is the first release prepared after the [acceptance of cert-manager into the CNCF sandbox](https://blog.jetstack.io/blog/cert-manager-cncf/), the copyright strings have been changed to remove references to Jetstack.
+
+* The `User-Agent` request header has changed from `jetstack-cert-manager/<version>` to `cert-manager/<version>`.
+
+# New Features
+
+## Additional options for cert-manager controllers
+
+* The cert-manager controller can now be configured to expose profiling information using the new `--enable-profiling` flag.
+
+* cainjector leader election leases are now customizable using the new flags `--leader-election-lease-duration`, `--leader-election-renew-deadline` and `--leader-election-retry-period`.
+
+## Usability improvements
+
+* cert-manager can now create Java KeyStores that are compatible with Java 8 or greater. A file named `keystore.jks` will be added to the secret specified in the `Certificate.spec.secretName` encrypted with the password specified in the `Certificate.spec.jks.passwordSecretRef` secret.
+
+  ```yaml
+  apiVersion: cert-manager.io/v1
+  kind: Certificate
+  metadata:
+    name: jks-example
+  spec:
+    secretName: jks-keystore
+    jks:
+      create: true
+      passwordSecretRef:
+        name: supersecret
+        key: password
+  ```
+
+* ingress-shim now supports the new `cert-manager.io/usages` annotation for specifying custom key usages. If this isn't set, it now defaults to `server auth`, but a comma separated list of [any valid usages](https://pkg.go.dev/github.com/jetstack/cert-manager@v1.2.0/pkg/apis/certmanager/v1#KeyUsage) can be specified.
```

Comment:
It maintains the same key usage default of Digital Signature and KeyEncipherment 

https://github.com/jetstack/cert-manager/blob/969b678f330c68a6429b7a71b271761c59651a85/pkg/apis/certmanager/v1/types.go#L209

Reply:
👍

Reply:
Oh, it's never server-auth. I was mislead by the PR description